### PR TITLE
Release notes for Foreman 3.0.0-RC2

### PR DESCRIPTION
**Pull request by upadhyeammit**

### CODE REVIEW

File: _includes/manuals/3.0/1.2_release_notes.md [hunk 1]
```diff
@@ -35,216 +35,230 @@ Fact parsers for Ansible, Chef, Salt and Subscription Manager were moved to the
 
 ### Release notes for 3.0.0
 #### Foreman
-* Make docs.theforeman.org the official Foreman and Katello documentation space ([#33169](https://projects.theforeman.org/issues/33169))
```
Comment (evgeni) [inline _includes/manuals/3.0/1.2_release_notes.md:38]: feels like things were removed here but not updated?

Comment (upadhyeammit) [reply _includes/manuals/3.0/1.2_release_notes.md:38]: As discussed due to fixes in categories notes have updated. @evgeni any more suggestions?

Comment (evgeni) [reply _includes/manuals/3.0/1.2_release_notes.md:38]: I had no further comments (but also didn't look too closely)

**Comment by upadhyeammit** [comment]
@evgeni thanks for review, merging notes!

**Pull request merged**